_pages/en_US/bannerbomb3.txt

```diff
@@ -27,38 +27,45 @@ If you have a Taiwanese console (indicated by a T after the system version), fol
+1. Copy `bb3.bin` from `DSIWARE_EXPLOIT.zip` to the root of your SD card
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
 1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` on your SD card
   + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
   + `<ID1>` is a 32-letter folder inside of the `<ID0>`
   ![]({{ "/images/screenshots/bb3/dsiware-location-1.png" | absolute_url }}){: .notice--info}
 1. Create a folder named `Nintendo DSiWare` inside of the `<ID1>`
   + If you already had the folder *and* there are any existing DSiWare backup files (`<8-character-id>.bin`) inside, copy them to your PC and remove them from your SD card
-1. Copy the `F00D43D5.bin` file from `BannerBomb3.zip` to the `Nintendo DSiWare` folder
+1. Copy the `F00D43D5.bin` file from `DSIWARE_EXPLOIT.zip` to the `Nintendo DSiWare` folder
 
 ![]({{ "/images/screenshots/bb3/dsiware-location-2.png" | absolute_url }})
 {: .notice--info} 
 
-#### Section II - BannerBomb3
+#### Section II - Hardware Button Check
 
-In this section, you will trigger the BannerBomb3 exploit using the DSiWare Management menu and copy the resulting file dump to your computer so that you can use it on the next page.
+In this section, you will test the SAFE_MODE function of your device. This will determine which method you will follow in the next section.
 
-1. Reinsert your SD card into your device
-1. Power on your device
-1. Launch System Settings on your device
-1. Navigate to `Data Management` -> `DSiWare` -> `SD Card` ([image](/images/screenshots/bb3/dsiware-management.png))
-  + Your system should flash Magenta (pink/purple) and then crash a few seconds later. This means it worked
+1. With your device still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), and while holding these buttons together, power on your device
+  + Keep holding the buttons until the device boots into Safe Mode (a "system update" menu)
+1. If prompted to update, press Cancel
+  + If the device boots to the HOME Menu, just continue to the next step
 1. Power off your device
-1. Insert your SD card into your computer
-1. You should now have `42383841.bin` on the root of your SD card. Copy this file to somewhere on your computer, as you will use it on the next page
-1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare` on your SD card
-1. Delete `F00D43D5.bin` from your Nintendo DSiWare folder and from your computer. This file will not be needed anymore
 
+___
+
+### Next steps: Choose an exploit
+
+If your device booted into Safe Mode and prompted to you to do a system update in Section II, Safe Mode is working on your device.
 
-Continue to [Installing boot9strap (Fredtool)](installing-boot9strap-(fredtool))
+The next steps are dependent on whether Section II booted into the SAFE_MODE successfully.
```

Comment:
This line is redundant as the same information is communicated in the below infoboxes